crates/store/re_video/src/demux/mod.rs

```diff
@@ -96,8 +90,40 @@ impl VideoData {
 
     /// The codec used to encode the video.
     #[inline]
-    pub fn codec(&self) -> &str {
-        &self.config.codec
+    pub fn human_readable_codec_string(&self) -> String {
+        let mut string = self.config.stsd.contents.codec_string().unwrap_or_default();
+
+        match &self.config.stsd.contents {
+            re_mp4::StsdBoxContent::Av01 { .. } => {
+                string += " (AV1)";
+            }
+            re_mp4::StsdBoxContent::Avc1 { .. } => {
+                string += " (H.264)";
+            }
+            re_mp4::StsdBoxContent::Hvc1 { .. } => {
+                string += " (H.265 HVC1)";
+            }
+            re_mp4::StsdBoxContent::Hev1 { .. } => {
+                string += " (H.265 HEV1)";
+            }
+            re_mp4::StsdBoxContent::Vp08 { .. } => {
+                string += " (VP8)";
+            }
+            re_mp4::StsdBoxContent::Vp09 { .. } => {
+                string += " (VP9)";
+            }
+            re_mp4::StsdBoxContent::Mp4a { .. } => {
+                string += " (AAC)";
+            }
+            re_mp4::StsdBoxContent::Tx3g { .. } => {
+                string += " (TTXT)";
+            }
+            re_mp4::StsdBoxContent::Unknown(four_cc) => {
+                string += &format!(" ({four_cc})");
```

Comment:
afaik the String is already the four_cc in this case, so maybe rather go `Unkown ({string})`
(begs the question if this kind of soft-knowldege-dependency is a very-bad-thing)